package.json

```diff
@@ -35,14 +37,15 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@digitalcredentials/bitstring": "^2.0.1",
-    "@digitalcredentials/vc": "^4.1.1",
     "@digitalbazaar/vc-status-list-context": "^3.0.1",
+    "@digitalcredentials/bitstring": "file:../bitstring",
```

Comment:
**BLOCKING:** Update `bitstring` dependency after a new NPM release for [this PR](https://github.com/digitalcredentials/bitstring/pull/4) has been deployed.